addons/binding/org.openhab.binding.vitotronic/ESH-INF/binding/binding.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <binding:binding id="vitotronic"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
-        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
Travis runs every time you push a new commit - as I can see, all seems to be green now.